packages/grafana-data/src/types/icon.ts

```diff
@@ -166,9 +166,12 @@ export const availableIconsIndex = {
   kubernetes: true,
   'layer-group': true,
   'layers-alt': true,
+  'legend-show': true,
+  'legend-hide': true,
   'library-panel': true,
   'line-alt': true,
   link: true,
+  'link-broken 1': true,
```

Comment:
```suggestion
  'link-broken-1': true,
```
Is this possible for consistency? Or is the 1 needed at all?

Reply:
is there any reason why it is 1? What about `link-broken` ?

Reply:
There's already a [link-broken icon ](https://github.com/grafana/grafana/blob/main/public/img/icons/unicons/link-broken.svg). It looks the same so I'll just use this instead